Common/tests/Helpers/TestDatabaseProvider.cs

```diff
@@ -57,8 +57,18 @@ public TestDatabaseProvider(Action<IServiceCollection>? configurator = null)
                     StandardErrorLogger     = line => logger.LogError(line),
                   };
 
+    var loggerProvider = new ConsoleForwardingLoggerProvider();
+    var loggerDB       = loggerProvider.CreateLogger("db commands");
+
     runner_ = MongoRunner.Run(options);
-    client_ = new MongoClient(runner_.ConnectionString);
+    var settings = MongoClientSettings.FromConnectionString(runner_.ConnectionString);
+    //settings.ClusterConfigurator = cb =>
+    //                               {
+    //                                 cb.Subscribe<CommandStartedEvent>(e => loggerDB.LogTrace("{CommandName} - {Command}",
+    //                                                                                          e.CommandName,
+    //                                                                                          e.Command.ToJson()));
+    //                               };
```

Comment:
If you don't need it, remove it.

Reply:
I don't want to see the requests most of the time but I needed it today.

Reply:
Maybe we could have this option more globally. Anyway, approving.